pom.xml

```diff
@@ -119,7 +129,7 @@
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>findbugs-maven-plugin</artifactId>
-        <version>3.0.0</version>
```

Comment:
Can be removed, the new plugin POM defines the build and test process. It also uses SpotBugs now